RobotCrawler/src/main/java/parser/ebookscom/BookDataFactory.java

```diff
@@ -0,0 +1,73 @@
+package parser.ebookscom;
+
+
+
+import model.BookData;
```

Comment:
Not yet but, one of the tables in the database is book so I reserved this name for that